junebug/amqp.py

```diff
@@ -0,0 +1,115 @@
+from twisted.internet.defer import inlineCallbacks, returnValue, Deferred
+from twisted.internet.protocol import ReconnectingClientFactory
+from twisted.python import log
+from txamqp.client import TwistedDelegate
+from txamqp.content import Content
+from txamqp.protocol import AMQClient
+from vumi.utils import vumi_resource_path
+from vumi.service import get_spec
+
+
+class AmqpFactory(ReconnectingClientFactory, object):
```

Comment:
Update: we are probably going to change vumi's amqp things to do what we need instead of doing the current amqp things we are doing in junebug.